src/heat_solve.H

```diff
@@ -0,0 +1,29 @@
+#ifndef HEAT_SOLVE_H
+#define HEAT_SOLVE_H
+
+#include <vector>
+
+namespace openturbine::heat_solve {
```

Comment:
I am assuming that this code will not be sticking around once we have OpenTurbine code.  If it will be kept long term, it needs significant work to bring it up to a modern, sustainable standard (for example, the use of raw pointers and malloc is unnecessary in this context).  Let me know if you need help in this.

Reply:
Definitely not sticking around. It's a toy problem in order to demonstrate the unit test infrastructure.